src/components/Signup/Signup.js

```diff
@@ -74,11 +74,16 @@ export default injectIntl(function({ intl }) {
       errors[fieldNames.email] = _('validation_messages.error_invalid_email_address');
     }
 
+    const digitRegex = /[0-9]/;
     if (!values[fieldNames.password]) {
-      // TODO: I think that password validation has a different format
-      errors[fieldNames.password] = _('validation_messages.error_required_field');
-    } else {
-      // TODO: validate password
+      errors[fieldNames.password] = { empty: true };
+    } else if (values[fieldNames.password].length < 8) {
+      errors[fieldNames.password] = { charLength: true };
+      if (!digitRegex.test(values[fieldNames.password])) {
+        errors[fieldNames.password] = { charLength: true, digit: true };
+      }
+    } else if (!digitRegex.test(values[fieldNames.password])) {
+      errors[fieldNames.password] = { digit: true };
```

Comment:
I understand the reason for having it in _validate_ function (or the same, in the _validation_ helper that @fcoronelmakingsense is creating), but I am still not comfortable in having this so far from _PasswordWrapper_ because both things are really coupled.

But, I am not sure about how to improve it, using _ref_ is an option, but I do not like it either. Maybe we should learn more about _Formik_ for further releases.

Reply:
It would be great to have this validation in form helper, since it's small, and as you've said highly coupled with display in PasswordWrapper. I will try adding it there.

Reply:
We have talked about it. Resolving the conversation...